alembic/env.py

```diff
@@ -0,0 +1,81 @@
+from logging.config import fileConfig
```

Comment:
The import statement on this line is incorrect. The correct syntax should be `from logging.config import fileConfig` without the slash (`/`).